build/features.py

```diff
@@ -371,7 +371,8 @@ def sources(self, build):
 
 
 class Vamp(Feature):
-    INTERNAL_LINK = False
+    # Always use the internal instead of any pre-installed version to
+    # avoid unexpected behavior.
```

Comment:
This may violate distributions' packaging guidelines. How about making it use the internal version if the system version is older than the internal version?